src/query/v2/shard_request_manager.hpp

```diff
@@ -281,6 +283,42 @@ class ShardRequestManager : public ShardRequestManagerInterface {
     return responses;
   }
 
+  std::vector<CreateExpandResponse> Request(ExecutionState<CreateExpandRequest> &state,
+                                            std::vector<NewExpand> new_edges) override {
+    MG_ASSERT(!new_edges.empty());
+    MaybeInitializeExecutionState(state, new_edges);
+    std::vector<CreateExpandResponse> responses;
+    auto &shard_cache_ref = state.shard_cache;
+    size_t id{0};
```

Comment:
```suggestion
    int64_t id{0};
```

Reply:
Why should this be `int64_t`? It is only used to index into arrays and increment it. For that a `size_t` is more appropriate in this case (yes, having the size type as unsigned in a big mistake in C++, but in this case a size_t conforms the API more than an `int64_t`).